src/MockPlayerBridge.ts

```diff
@@ -41,23 +73,157 @@ export class MockPlayerBridge extends AbstractPlayerBridge {
     return 'dummy-stream-id';
   }
 
-  protected subscribeFullscreen(listener: (value: boolean) => void) {
-    // Note: LiveryPlayer only listens to itself becoming fullscreen; not just anything
-    document.addEventListener('fullscreenchange', () => {
-      listener(!!document.fullscreenElement);
+  protected pause() {
+    this.setPlaybackState('PAUSED');
+  }
+
+  protected play() {
+    this.setPlaybackState('PLAYING');
+  }
+
+  protected reload() {
+    this.setPlaybackState('PLAYING');
+  }
+
+  protected seek(position: number) {
+    this.setPlaybackState('SEEKING');
+    return position;
+  }
+
+  protected selectQuality(index: number) {
+    return index;
+  }
+
+  protected setControlsDisabled(disabled: boolean) {
+    return disabled;
+  }
+
+  protected setDisplay(display: DisplayMode) {
+    return this.setDisplayState(display);
+  }
+
+  protected setMuted(muted: boolean) {
+    return this.setMutedState(muted);
+  }
+
+  protected submitUserFeedback(payload: UserFeedbackPayload) {
+    return payload;
+  }
+
+  protected subscribeConfig(listener: (value: Config) => void) {
+    const config: Config = {
+      controls: {
+        cast: true,
+        contact: true,
+        error: true,
+        fullscreen: true,
+        mute: true,
+        pip: true,
+        play: true,
+        quality: true,
+        scrubber: true,
+      },
+      customerId: 'id',
+      streamPhase: 'PRE',
+      streamPhases: ['LIVE', 'POST', 'PRE'],
+      tenantId: 'id',
+    };
+
+    setTimeout(() => {
+      const newConfig = { ...config };
+      newConfig.controls = {
+        ...newConfig.controls,
+        cast: false,
+        scrubber: false,
+      };
+      newConfig.streamPhase = 'LIVE';
+      listener(newConfig);
+    }, 1500);
+    setTimeout(() => {
+      const newConfig = { ...config };
+      newConfig.controls = {
+        ...newConfig.controls,
+        cast: true,
+        scrubber: true,
+      };
+      newConfig.streamPhase = 'POST';
+      listener(newConfig);
+    }, 3000);
+
+    listener(config);
+    return config;
+  }
+
+  protected subscribeDisplay(listener: (value: DisplayMode) => void) {
+    document.addEventListener(MockListenerEvents.DISPLAY, () => {
```

Comment:
As mentioned above this should not use DOM events. Instead just keep track of a list of `displayListeners` and notify those from `setDisplay()`.